Make.config

```diff
@@ -355,7 +354,8 @@ JENKINS_RESULTS_DIRECTORY ?= $(abspath $(TOP)/jenkins-results)
 # Clone files instead of copying them on APFS file systems. Much faster.
 CP:=$(shell df -t apfs / >/dev/null 2>&1 && echo "cp -c" || echo "cp")
 
-XCODE_ARCHIVE_VERSION=xcode-$(XCODE_PRODUCT_BUILD_VERSION)
+# WORKAROUND, should be  xcode-$(XCODE_PRODUCT_BUILD_VERSION) but we are stuck due to xcode 11.4 beta 1 issue: https://github.com/xamarin/xamarin-macios/issues/7872
+XCODE_ARCHIVE_VERSION=xcode-11C29
```

Comment:
Do we need this still?? @akoeplinger maybe you do know?

Reply:
Got confirmation from @akoeplinger online, looks good.

Reply:
Yes, still needed.